msteams-platform/bots/how-to/conversations/request-headers-of-the-bot.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Send tenant ID and conversation ID to the request headers of the bot
+description: describes how to send tenant ID and conversation ID to the request headers of the bot.
+ms.topic: conceptual
+---
+
+# Send tenant ID and conversation ID to the request headers of the bot
+
+The current outgoing requests to the bot do not contain in the header or URL any information that helps bots route the traffic without unpacking the entire payload. The activities are sent to the bot to a URL similar to https://va.ai.dynamics.com/api/v1/messages/bots/1f0fe267-c3e3-4011-85fa-xxxxxxxxxx. Requests are received to show the conversation ID and tenant ID in the headers.
```

Comment:
can we change please the URL to be more generic "https://<your_domain>/api/messages"?